packages/code-studio/src/log/LogExport.ts

```diff
@@ -72,9 +72,11 @@ function getReduxDataString() {
 
 /**
  * Export support logs with the given name.
+ * @param metadata Any extra metadata to be written to a metadata file
  * @param fileNamePrefix The zip file name without the .zip extension. Ex: test will be saved as test.zip
  */
 export async function exportLogs(
+  metadata = {},
```

Comment:
Seems a little strange to require the metadata to be passed in separately, when everything else is pulled from redux/global stores here. Why not just have a `getMetadata()` function defined in LogExport, then the SettingsMenu doesn't need to change as well?

Reply:
My thought on this was that the log export doesn't need to be duplicated in Enterprise. It could be part of some shared package. Then Enterprise can add other metadata but the logic for saving the logs is the same.

Might not work though if we add more Enterprise specific data to the logs

Reply:
I think there will always be different stuff - I could see us logging console sessions in both, but perhaps the list of PQs in the active dashboard and their statuses or something like that. I think we had a bunch of ideas on the Enterprise ticket.
In other words, I think LogExport is unique to each and there may be a bit of overlap but not enough to try and share design.

Reply:
Alternatively, pass in all the files to export, eg:
```
exportZip({
  'metadata.json': { uiVersion, userAgent },
  'console.txt': logHistory.getFormattedHistory(),
  'redux.json': getReduxDataString(),
})
```
and then that iterates through all properties and creates the files, checking the type, if it's an object then do the safe stringify thing, etc.

Reply:
Just put it all in LogExport for now